lib/src/services/configuration-service.dart

```diff
@@ -1,18 +1,31 @@
 part of 'tmdb-service.dart';
 
-abstract class ConfigurationService {
+abstract class ConfigurationService with ResilientService {
   static const String _configPath = "3/configuration";
 
   Configuration _configuration;
   String _apiKey;
 
-  List<Country> _countries;
+  MovieService _movie;
```

Comment:
I'm sorry if I misled you with my previous comments, but I wasn't proposing to move the Movie/Tv services out of the TmdbService, especially not to ConfigurationService as it kind of don't feel right here.
As an idea maybe move them back to TmdbService where you could override the configuration setter to update Movie/TV service when necessary.